compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cpp

```diff
@@ -494,9 +494,6 @@ unsigned struct_ElfW_Phdr_sz = sizeof(Elf_Phdr);
   unsigned struct_input_id_sz = sizeof(struct input_id);
   unsigned struct_mtpos_sz = sizeof(struct mtpos);
   unsigned struct_rtentry_sz = sizeof(struct rtentry);
-#if SANITIZER_GLIBC || SANITIZER_ANDROID
-  unsigned struct_termio_sz = sizeof(struct termio);
-#endif
```

Comment:
I believe this section needs to stay because of Android, right?

Reply:
I was going to do that, but then I noticed that all the other users of the variable were guarded by `SANITIZER_GLIBC` only, so I thought maybe it wasn't needed at all on android.

Reply:
(on Android, <termio.h> is just `#include <termios.h>` anyway)